workflow/util/util.go

```diff
@@ -362,17 +363,50 @@ func ResumeWorkflow(wfIf v1alpha1.WorkflowInterface, workflowName string, nodeFi
 				workflowUpdated = true
 			}
 
+			nodes := wf.Status.Nodes
+			if wf.Status.IsOffloadNodeStatus() {
+				if !repo.IsEnabled() {
+					return false, fmt.Errorf(sqldb.OffloadNodeStatusDisabledWarning)
```

Comment:
is this warning or error?

Reply:
Error. If you're not able to get the offloaded nodes to modify them, how can you resume the workflow?

Reply:
maybe we should rename `OffloadNodeStatusDisabledWarning` to `OffloadNodeStatusDisabled` status? or maybe we should provide a sentinel error to use?

Reply:
I would support renaming to `OffloadNodeStatusDisabled`.

How would the sentinel error work? Not sure how it would be use here.

Reply:
Renamed to `OffloadNodeStatusDisabled`